lib/octokit/client/organizations.rb

```diff
@@ -595,6 +595,89 @@ def remove_organization_membership(org, options = {})
         user = options.delete(:user)
         user && boolean_from_response(:delete, "orgs/#{org}/memberships/#{user}", options)
       end
+
+      # Initiates the generation of a migration archive.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @option options [Array<String>] :repositories Repositories for the organization.
```

Comment:
You know what, I'm going to leave it as NWO. This is how the `.create_team` method prefers it too: https://github.com/octokit/octokit.rb/blob/1e8b395ba14a6d5d25b7dbe82a6a22a42ce4c585/lib/octokit/client/organizations.rb#L241-L246

Will definitely move it in as a second parameter though. `.create_team` can leave it in the options because it's not required. (I suspect I just stole the idea from this method entirely.)